src/lsg/patterns/property-items/enum-item/index.tsx

```diff
@@ -18,22 +19,39 @@ export interface EnumItemProps {
 	values: Values[];
 }
 
+export interface StyledIconProps {
+	open?: boolean;
+}
+
 const StyledEnumItem = styled.div`
 	width: 100%;
 `;
 
+const StyledIcon = styled(Icon)``;
+
 const StyledSelect = styled.select`
 	appearance: none;
-	border: none;
-	border-radius: 0;
-	font-size: 1em;
+	display: block;
+	box-sizing: border-box;
 	width: 100%;
+	padding: ${getSpace(Size.XS)}px;
+	margin-bottom: ${getSpace(Size.M)}px;
+
+	border: 0.5px solid ${colors.grey90.toString()};
+	border-radius: 3px;
+
+	background-color: ${colors.white.toString()};
+	background-image: url("data:image/svg+xml;utf8,<svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 24 24' fill='${colors.grey60.toString()}'><path d='M8,4 l8,8 -8,8 z' /></svg>");
```

Comment:
I added the icon as a background image (still wrong rotation) for now, as I haven't found a solution on how to import the icon component for this issue. Is there any way to resolve this @Palmaswell @LKuechler ?